versions.json

```diff
@@ -40,18 +40,41 @@
         "3.4.1"
       ]
     },
+    {
+      "path": "3.4.1/scala2.13-java11-python3-ubuntu",
+      "tags": [
+        "3.4.1-scala2.13-java11-python3-ubuntu"
+      ]
+    },
     {
       "path": "3.4.1/scala2.12-java11-r-ubuntu",
       "tags": [
         "3.4.1-scala2.12-java11-r-ubuntu",
         "3.4.1-r"
       ]
     },
+    {
+      "path": "3.4.1/scala2.13-java11-r-ubuntu",
+      "tags": [
+        "3.4.1-scala2.13-java11-r-ubuntu"
+      ]
+    },
     {
       "path": "3.4.1/scala2.12-java11-ubuntu",
       "tags": [
         "3.4.1-scala2.12-java11-ubuntu",
-        "3.4.1-scala"
+        "3.4.1-scala2.12",
+        "scala2.12",
+        "3.4.1-scala",
+        "scala"
+      ]
+    },
+    {
+      "path": "3.4.1/scala2.13-java11-ubuntu",
+      "tags": [
+        "3.4.1-scala2.13-java11-ubuntu",
+        "3.4.1-scala2.13",
+        "scala2.13"
```

Comment:
Would we want to do the scala 2.13 tag with 3.5 instead  @Yikun

Reply:
Yes, agree! As I mentioned above comment, 2.13 better to introduce in latest versio (3.5.0 for now)

https://github.com/apache/spark-docker/pull/52#pullrequestreview-1676547189